### PR TITLE
chore: updated cosmwasm deps

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

**Comment by dzmitry-lahoda** [comment]
building '/nix/store/0ik1fl7brsa8frgkm3nrc9d2d556r7wq-beaker-0.1.8.drv'...
error (ignored): error: cannot unlink '/tmp/nix-build-hermes-v1.7.4.drv-0/source/target/x86_64-unknown-linux-gnu/release/deps': Directory not empty
error (ignored): error: cannot unlink '/tmp/nix-build-beaker-0.1.8.drv-0': Directory not empty
error: writing to file: No space left on device
Error: Process completed with exit code 1.

File: lib/default.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@ nix-std: {
   buildCosmwasmContract = let
     target = "wasm32-unknown-unknown";
     # from https://github.com/CosmWasm/rust-optimizer/blob/main/Dockerfile
-    rust = pkgs.rust-bin.stable."1.70.0".default.override {
+    rust = pkgs.rust-bin.stable."1.75.0".default.override {
```
Comment (JonathanLorimer) [inline lib/default.nix:8]: Does this need a pinned rust version, or could we just use nightly or `pkgs.rust-bin.stable.latest.default`?

**Review by JonathanLorimer — APPROVED** [review]
Looks great, just one question about rust versions.

File: lib/default.nix (hunk 1 above)
Comment (dzmitry-lahoda) [reply lib/default.nix:8]: yes. wasm has very subtle build and runtime ssues regarding wasm features.
the best option to build with exactly same version official docker container builds.

**Comment by JonathanLorimer** [comment]
hopefully this will help get CI passing https://github.com/informalsystems/cosmos.nix/pull/227

**Pull request merged**